assets/queries/dockerfile/apt_get_install_pin_version_not_defined/test/positive2.dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM busybox4
```

Comment:
Can you add a negative sample with `;` for `apt_get_install_pin_version_not_defined`, please?